cluster/gce/windows/README-GCE-Windows-kube-up.md

```diff
@@ -95,6 +95,44 @@ To teardown the cluster run:
 PROJECT=${CLOUDSDK_CORE_PROJECT} KUBERNETES_SKIP_CONFIRM=y ./cluster/kube-down.sh
 ```
 
+#### 2b. Create a Kubernetes end-to-end (E2E) test cluster	
```

Comment:
I'm not sure that these headings are accurate:

\#### 2a. Create a regular Kubernetes cluster
\#### 2b. Create a Kubernetes end-to-end (E2E) test cluster

Is there any difference in the resulting clusters? Can e2e tests be run against clusters brought up using the 2a steps?

I'm wondering if this should instead be:

\#### 2. Create a Kubernetes cluster
...explain that there are two ways that work, and explain what the difference between them is...
\##### Using `kube-up`
\##### Using `kubetest`

What do you think?

Reply:
Agree, did some experiments, the main differences are 
1. Both e2e test clusters and e2e test scripts are setup using the config `cluster/gce/config-test.sh`, so the test scripts look for the e2e cluster only and run tests.
2. Only e2e test clusters enables SSH on windows nodes. Regular clusters are not. So couldn't dump logs thru scp on regular cluster windows node.

Added explanations.

Reply:
Also the differences between `kubetest  --up` & `./hack/e2e-internal/e2e-up.sh` is that kubetest has extra step at the end: `./cluster/kubectl.sh --match-server-version=false version` which seems just update the kubectl binary. And the kubectl.sh will be deprecated soon from the file descriptions.